tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/auth/AuthnWithExpiredCertificateTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is this newly added test class? If so shall we change do above?
```suggestion
 *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```